script.js

```diff
@@ -0,0 +1,144 @@
+const todoForm = document.querySelector('.inputForm');
+const todoInput = todoForm.querySelector('input');
+const todoList = document.querySelector('.toDos');
+const doneList = document.querySelector('.dones');
+
+const saveList = 'savedList';
+let savedList = [];
+
+const saveFinishedList = 'savedFinList';
+let savedFinList = [];
+
+loadList();
+
+// To get the input from the form by adding an
+// event listener for submit button
+todoForm.addEventListener('submit', function (e) {
+  e.preventDefault();
+
+  const input = this.inputText;
+  const inputValue = input.value;
+  // if empty, neglect it
+  if (inputValue != '') {
+    addTask(inputValue);
+    saveTasks(inputValue);
+  }
+  //when submitted, reset the input form
+  todoForm.reset();
+});
+
+function saveFinTasks(todo) {
+  savedFinList.push(todo);
+  localStorage.setItem(saveFinishedList, JSON.stringify(savedFinList));
+}
+
+function saveTasks(todo) {
+  savedList.push(todo);
+  localStorage.setItem(saveList, JSON.stringify(savedList));
+}
+
+function loadList() {
+  const loadedList = localStorage.getItem(saveList);
+  if (loadedList !== null) {
+    const tempList = JSON.parse(loadedList);
+
+    for (let i of tempList) {
+      addTask(i);
+      saveTasks(i);
+    }
```

Comment:
js의 array method를 적극 활용해봅시다!
```suggestion
    tempList.forEach((todoItem) => {
        addTask(todoItem);
        saveTasks(todoItem);
    })
```

Reply:
앗 저도 이 부분 말씀드리려고 했는데 선종님께서 말씀해주셨네요! for in 문은 js에서 굉장히 고전적인 형식이라서 array method를 활용해보면 좋을것 같습니다!